File: 오선민/programmers/P12912.py
'''
문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/12912
시간: 987.32ms
메모리: 10.2MB
PR 제목 : 날짜 사이트_명 문제_번호 사용_언어
'''

def solution(a, b):
    result = 0
    if a > b:
        a, b = b, a
    for i in range(a, b+1):
        result += i
    return result

File: 강재연/programmers/P12912.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/12912
// 시간: 23.81 ms
// 메모리: 36.3 MB

function solution(a, b) {
    // 정렬이 되어있지 않기 때문에 혹시 a > b 이면 순서 바꿔줌
    if(a > b) {
        let temp = a;
        a = b;
        b = temp;
    }

    // 누적합 구하기
    let sum = 0;
    for(i=a; i<=b; i++){
        sum += i;
    }
    return sum;
}
